src/coreclr/vm/mono/mono_coreclr.cpp

```diff
@@ -69,26 +71,29 @@ typedef intptr_t ManagedStringPtr_t;
 // Generated by UnityEmbedHost.Generator - Commit these changes
 struct HostStruct
 {
+    int32_t (*add_system_assembly_search_path)(const char* raw_path);
     int (*array_length)(MonoArray* array);
     MonoArray* (*array_new)(MonoDomain* domain, MonoClass* klass, guint32 n);
     MonoObject* (*assembly_get_object)(MonoDomain* domain, MonoAssembly* assembly);
     MonoAssembly* (*assembly_loaded)(MonoAssemblyName* aname);
     MonoClass* (*class_from_name)(MonoImage* image, const char* name_space, const char* name, gboolean ignoreCase);
     MonoClass* (*class_from_systemtypeinstance)(MonoObject* systemTypeInstance);
     MonoImage* (*class_get_image)(MonoClass* klass);
+    MonoDomain* (*create_assembly_load_context)(const char* raw_name);
     MonoException* (*exception_from_name_msg)(MonoImage* image, const char* name_space, const char* name, const char* msg);
     MonoReflectionField* (*field_get_object)(MonoDomain* domain, MonoClass* klass, MonoClassField* field);
     intptr_t (*gchandle_get_target_v2)(intptr_t handleIn);
     uintptr_t (*gchandle_new_v2)(MonoObject* obj, gboolean pinned);
     uintptr_t (*gchandle_new_weakref_v2)(MonoObject* obj, gboolean track_resurrection);
     MonoImage* (*get_corlib)();
     MonoException* (*get_exception_argument_null)(const char* arg);
+    MonoMethod* (*get_method)(MonoImage* image, guint32 token, MonoClass* klass);
+    MonoDomain* (*get_system_assembly_load_context)();
     const char* (*image_get_filename)(MonoImage* image);
     const char* (*image_get_name)(MonoImage* image);
-    int32_t (*image_get_table_rows)(MonoImage* image, int32_t table_id);
     MonoImage* (*image_loaded)(const char* name);
-    intptr_t (*load_assembly_from_data)(const char* data, int64_t size);
-    intptr_t (*load_assembly_from_path)(const char* path, int32_t length);
+    intptr_t (*load_assembly_from_data)(MonoDomain* rawAlc, const char* data, int64_t size);
```

Comment:
Neither those ones

Reply:
to clarify - are you suggesting to remove the `load_assembly_from_*` proxies or `MonoDomain` parameter?
note, that without ability to load the assembly into a new ALC we can't load it again and need to modify Editor assemblies loading logic (and AssetDB refresh)

Reply:
Yes, this is part of the next step the Scripting Runtime team will be working on. For now, the editor should "straight load" assemblies and never unload them, and the Scripting Runtime team will work on the proper loader, in managed code :)